src/consensus/aft/raft_types.h

```diff
@@ -190,5 +192,14 @@ namespace aft
   DECLARE_JSON_TYPE_WITH_BASE(RequestVoteResponse, RaftHeader);
   DECLARE_JSON_REQUIRED_FIELDS(RequestVoteResponse, term, vote_granted);
 
+  struct ProposeRequestVote : RaftHeader
+  {
+    // Message sent by a leader stepping down after observing its own retirement
+    // being committed.
+    Term term;
```

Comment:
I don't think this comment belongs here, as it's so precise its likely to rot quickly (we tweak the conditions on sending, or send in some additional case*).

If this definition needs a justification (because its not a "standard" message type), it should be something more general: "A node sends this to nudge another node to begin an election, for instance because the sender is a retiring primary"

\* Expanding on this slightly, but conjecture for now: I've got an idea, untested, that a slight extension of `RequestVoteResponse` would let a _failed_ election discover a _viable_ candidate. People vote against you because you're illegitimate; with a majority saying "no _because I'm at `X`_" you've got enough information to pick a node that a majority would think is legitimate (albeit on stale information). Prodding that node to become candidate immediately is preferable to waiting for a timeout.

Reply:
Ok, I'll simplify the comment.